### PR TITLE
Added homework now it is tested

### PR DESCRIPTION
**Pull request by neveenatik**
Hi George and Alex,

Homework is not ready for review unfortunately. Very sorry about that. I got stuck but I will get there.
Thanks for your patience.

### CODE REVIEW

**Comment by GeorgeSapkin** [comment]
Is this ready for a review?

**Comment by neveenatik** [comment]
Hi George,
Homework is finally ready for review. Thanks a lot for your patience! Please accept my apology again for being late.

**Comment by neveenatik** [comment]
Very good question. I think I was thinking about the requirement of the homework rather than the app ☺️ 
I added that now.
Thanks George for your feedback. I really appreciate that!

File: week3/homework/src/index.js [hunk 1]
```diff
@@ -1,3 +1,97 @@
...
+app.delete('/todos/:id', (req, res, next) => {
+  const { id } = req.params;
+  deleteToDo(id)
+    .then(readToDos)
+    .then(toDoList => res.send(toDoList))
+    .catch(err => next(err));
+});
+
+// resetTodos
+app.delete('/reset', (req, res, next) => {
```
Comment (GeorgeSapkin) [inline week3/homework/src/index.js]: I think you've meant this to be `/todos` instead of `/reset`.

**Comment by neveenatik** [comment]
Hi George,
The route is changed to todos as requested.
Thanks for your review.

**Comment by neveenatik** [comment]
Hi George,

You are right. I thought if you update a todo then it should be reset to undone otherwise why does anyone update a done todo. 
I have made the requested changes.
Thanks again for your review.

**Pull request closed without merging**